content/blog/2021/02/2021-02-08-docker-base-os-upgrade.adoc

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Docker image updates"
+tags:
+- announcement
+- docker
+- platform-sig
+author: markewaite
+opengraph:
+  image: /images/docker/dockerJenkins-updated-os-2021-02.png
+---
+
+Beginning with Jenkins 2.279 and Jenkins 2.263.4, the Jenkins project is upgrading the base operating system and Java version used in the `jenkins/jenkins:latest` and `jenkins/jenkins:lts` images.
+The update includes the most recent AdoptOpenJDK 8u282 release and Debian 10 (link:https://www.debian.org/releases/buster/["Buster"]).
+The Java update includes fixes made in JDK 8u252, JDK 8u262, JDK 8u272, JDK 8u275, and JDK 8u282.
```

Comment:
```suggestion
The Java update includes changes made in JDK 8u252, JDK 8u262, JDK 8u272, JDK 8u275, and JDK 8u282.
```

Reply:
Do all the versions need listing? Seems a bit strange, non blocking